packages/utils/k3d/build.yaml

```diff
@@ -3,8 +3,7 @@ requires:
   category: "development"
   version: ">=0"
 prelude:
-{{$arch:=(default .Values.arch "amd64")}}
-
+{{$arch:=(default "amd64" .Values.arch)}}
```

Comment:
This is Spring right? http://masterminds.github.io/sprig/defaults.html

Reply:
yup! https://github.com/mudler/luet/blob/master/go.mod#L6